### ISSUE TITLE
[SUPPORT] I can't install it correctly

### ISSUE THREAD

**Issue by frapreviti** [flixlix/flixlix-cards#169]
Hi, I'm trying to install Power Flow Card Plus via HACS but I can't get it to work.

<img width="1367" alt="Screenshot 2024-11-19 alle 21 16 22" src="https://github.com/user-attachments/assets/d1ce0b76-3088-45cb-b967-8f07cfcd01e3">

After I click on Download, I am faced with the warning:

_Version v0.2.4 will be downloaded

Once downloaded, it will be found in '/config/www/community/power-flow-card-plus'
Once the download is complete, since you are not using Lovelace in archive mode, you need to manually add the asset with these settings:

url: /hacsfiles/power-flow-card-plus/power-flow-card-plus.js
type: module_

<img width="841" alt="Screenshot 2024-11-19 alle 21 19 33" src="https://github.com/user-attachments/assets/fbdb0429-b4e8-4d39-a878-a5e2e12e247a">

In reality in that indicated path, the file does not exist, so I look for the file and insert the correct one:

<img width="860" alt="Screenshot 2024-11-19 alle 21 26 46" src="https://github.com/user-attachments/assets/300a8031-f6f5-413b-a8f3-e4f1d389fad6">

Then insert the resource URL into the configuration file:
<img width="503" alt="Screenshot 2024-11-19 alle 21 28 45" src="https://github.com/user-attachments/assets/9628b6ec-4c47-4720-bc7e-1e4b0367cb2b">

Then I go to Settings > Devices > Power Flow Card but the screen that appears is this, without any possibility of configuration or addition to the Dashboard.

<img width="1354" alt="Screenshot 2024-11-19 alle 21 30 19" src="https://github.com/user-attachments/assets/e3cb432a-0153-41fa-8109-eb7b905fcbe6">

Home assistant Core on
Raspberry Pi 4

Thanks for the help!

Where am I wrong?

**Comment by flixlix** [comment]
after installing you should go to your dashboard and add a new card. then this should be in the end of the list. can you try this?

**Comment by flixlix** [comment]
closing this for now
